glossary.md

```diff
@@ -0,0 +1,25 @@
+### Defining Terms
+* Community
```

Comment:
If we use a low-level `####` heading instead, then the gh markdown parser will give us links for each item. Not sure if it's worth the effort though.